books/RayTracingInOneWeekend.html

```diff
@@ -864,11 +864,12 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [normals-point-against]: <kbd>[sphere.h]</kbd> Remembering the side of the surface]
 
-The decision whether to have normals always point out or to always point against the ray is based on
```

Comment:
What if we have a triangle.  Outward from the surface could go either way.  Against the incident ray seems like the only thing worth mentioning